lib/matplotlib/patches.py

```diff
@@ -942,18 +945,18 @@ def __init__(self, values, edges, *,
             The direction of the steps. Vertical means that *values* are along
             the y-axis, and edges are along the x-axis.
 
-        baseline : float or None, default: 0
-            Determines starting value of the bounding edges or when
+        baseline : float, 1D array-like or None, default: 0
```

Comment:
```suggestion
        baseline : float, array-like or None, default: 0
```
We usually don't specify this explicitly.